src/neo4j_genai/neo4j_queries.py

```diff
@@ -53,7 +53,9 @@
 def _get_hybrid_query() -> str:
     return (
         f"CALL {{ {VECTOR_INDEX_QUERY} "
-        f"RETURN node, score "
+        f"WITH collect({{node:node, score:score}}) AS nodes, max(score) AS max "
```

Comment:
perhaps rename to `vector_index_max_score` and `ft_index_max_score` to be more explicit? I think it makes the Cypher query more readable